api/manifest_util.go

```diff
@@ -71,7 +72,7 @@ func (x *IstioDistribution) Group() (string, error) {
 }
 
 func (x *IstioDistribution) IsOfficial() bool {
```

Comment:
this is intended to check "whether it has flavor in version string or not". So I think we do not need to change here, or maybe we should rename to `hasFlavor`.

Reply:
> question: what's the difference between `tetrate` and `istio` flavor?

At the moment there is no difference between tetrate and istio flavor except that the former is rebuilt in our pipeline. It is very likely tetrate flavor and and istio flavor can digress for a short while if patches are applied to tetrate as it may take some time in upstream to match the release cadence. Also, istio flavor eol is shorter when compared to tetrate and during that extended support duration, we are likely to add more patches while such a thing does not exist upstream.